src/Components/IncomeBlock/IncomeQuestion.js

```diff
@@ -174,7 +165,7 @@ const IncomeQuestion = ({
 
   const createIncomeStreamsDropdownMenu = (incomeStreamName, index) => {
     return (
-      <FormControl sx={{ m: 1, minWidth: 120, maxWidth: '100%' }} error={incomeStreamErrorController.showError}>
+      <FormControl sx={{ minWidth: 120, maxWidth: '100%' }} error={incomeStreamErrorController.showError}>
```

Comment:
Do you know what unit `minWidth: 120` is? i.e. is it `px`, `rem`, `em`, or `%`

Reply:
They're pixels.

Reply:
Can we change it to rem?